### PR TITLE
Add workspace service

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: api/v1/workspace_service.proto [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+service WorkspaceService {
+  rpc CreateNotebook(CreateNotebookRequest) returns (Notebook) {
+    option (google.api.http) = {
+      post: "/v1/workspaces/notebooks"
+      body: "*"
+    };
+  }
+  rpc ListNotebook(ListNotebooksRequest) returns (ListNotebooksResponse) {
```
Comment (kkaneda) [inline api/v1/workspace_service.proto:97]:
```suggestion
  rpc ListNotebooks(ListNotebooksRequest) returns (ListNotebooksResponse) {
```

File: api/v1/workspace_service.proto [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      get: "/v1/workspaces/notebooks/{id}"
+    };
+  }
+  rpc DeleteNotebook(DeleteNotebookRequest) returns (DeleteNotebookResponse) {
+    option (google.api.http) = {
+      delete: "/v1/workspaces/notebooks/{id}"
+    };
+  }
+
+  rpc StopNotebook(StopNotebookRequest) returns (StopNotebookResponse) {
```
Comment (kkaneda) [inline api/v1/workspace_service.proto:113]: Slightly prefer `Suspend`

**Review by kkaneda — APPROVED** [review]
LG!

File: api/v1/workspace_service.proto (hunk 2 above)
Comment (kenji-cloudnatix) [reply api/v1/workspace_service.proto:113]: Actually not sure which one is better. Leaving this as is looks good!

**Pull request merged**